File: output/CodeGen/codenet/C++/Python/s275739084.py
def main():
  A,B=map(int,raw_input().split())
  C=A+B
  if A%3!=0 and B%3!=0 and C%3!=0:
    print "Impossible"
  else:
    print "Possible"

if __name__=='__main__':
  main()

File: output/CodeGen/codenet/C++/Java/s275739084.java
public class s275739084 {
    public static void main(String[] args) {
        int A, B;
        Scanner in = new Scanner(System.in);
        A = in.nextInt();
        B = in.nextInt();
        int C = A + B;
        if (A % 3!= 0 && B % 3!= 0 && C % 3!= 0) {
            System.out.println("Impossible");
        } else {
            System.out.println("Possible");
        }
    }
}